keps/sig-storage/2268-non-graceful-shutdown/kep.yaml

```diff
@@ -0,0 +1,40 @@
+title: non graceful shutdown
+kep-number: 2268
+authors:
+  - "@yastij"
+  - "@xing-yang"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-node
+status: provisional 
```

Comment:
Is this ready to be marked implementable? Do we also want sig-node review on this?

Reply:
Changed to implementable.